deploy/crds/kubernetes/grafanadatasource.yaml

```diff
@@ -0,0 +1,317 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We shouldn't need this and our operator shouldn't own GrafanaDatasource CRD or create a grafana datasource.